packages/ckeditor5-media-embed/src/ui/mediaformview.js

```diff
@@ -71,6 +79,7 @@ export default class MediaFormView extends View {
 		 */
 		this.saveButtonView = this._createButton( t( 'Save' ), checkIcon, 'ck-button-save' );
 		this.saveButtonView.type = 'submit';
+		this.saveButtonView.bind( 'isEnabled' ).to( this, 'mediaURLInputValue' );
```

Comment:
```suggestion
		this.saveButtonView.bind( 'isEnabled' ).to( this, 'mediaURLInputValue', value => !!value );
```

I'd go with this here to make `Button.isEnabled` a boolean.